README.md

```diff
@@ -65,8 +70,10 @@ The Subscribe Window allows you to read and represent existing shapes on the net
 
 This menu provides the options present in the Publish Windows plus the following additions:
 
-* Time Based Filter: This value can be used to specify the minimum amount of time (in milliseconds) that the subscriber wants between updates. 
-* Content Based Filter: This filter draws a rectangle in the shapes window. Only the instances that are included in this rectangle are accepted. The rest of them are ignored. The user can dynamically resize and move this content filter. 
+* Time Based Filter: This value can be used to specify the minimum amount of time (in milliseconds) that the subscriber wants between updates.
+* Content Based Filter: This filter draws a rectangle in the shapes window. Only the instances that are included in this rectangle are accepted. The rest of them are ignored. The user can dynamically resize and move this content filter.
+
+Take into acoount that Disabling positive ACKs and modify Lifespan will not have any visual effect.
```

Comment:
I think `Deadeline` and `Lifespan` should also be described in this section, as these QoS are also applicable on the subscriber side. You can repeat the same as in the above section or mention that they work in a similar way